test/dotnet-new3.UnitTests/DotnetNewInstantiate.Approval.cs

```diff
@@ -307,5 +307,33 @@ public Task CanShowWarning_WhenHostDataIsIncorrect()
             });
         }
 
+        [Fact]
+        public Task CanShowError_OnTemplatesWithSameShortName()
+        {
+            string home = TestUtils.CreateTemporaryFolder("Home");
+            string workingDirectory = TestUtils.CreateTemporaryFolder();
+            string templateLocation = Helpers.InstallTestTemplate("Invalid/SameShortName", _log, home, workingDirectory);
```

Comment:
Just curious - why do we even allow to install conflicting templates? (or is it already disallowed and this is focused on a case of already installed conflict?)

Reply:
We considered this before, but as of the moment we consider https://github.com/dotnet/templating/issues/4135 a way to go.